src/main/java/org/folio/service/file/reader/LocalStorageCsvSourceReader.java

```diff
@@ -55,6 +65,8 @@ public boolean hasNext() {
     return iterator.next()
       .stream()
       .map(s -> s.replaceAll("\"", "").trim())
+      .map(s -> s.replaceAll(",", "").trim())
+      .filter(s -> s.matches(UUID_PATTERN))
```

Comment:
String matcher recompiles the regex each time, so it'll be a performance hit. Try to use a matcher